### ISSUE TITLE
[Feature] Move Docker install instructions to a common place versus under ChatQnA

### ISSUE THREAD

**Issue by mkbhanda** [opea-project/GenAIExamples#1042]
### Priority

Undecided

### OS type

Ubuntu

### Hardware type

Xeon-GNR

### Running nodes

Single Node

### Description

Docker is necessary for both building container images and for running it in a single node docker environment. We need to move it an expected place, that is common to all our build and run usages.
Currently it is under GenAIExamples/ChatQnA/docker_compose/install_docker.sh
Perhaps under GenAIInfra?

**Comment by srinarayan-srikanthan** [comment]
@feng-intel , yes we can move this out of chatqna. It was a one of thing for an event , so went in there. WE can move it outside chatqna

**Comment by yinghu5** [comment]
Remind: @xiguiw  please handle the issue

**Comment by chensuyue** [comment]
Maybe put in here, https://github.com/opea-project/docs/tree/main/guide/installation
